SH4RE/SH4RE/Assets.xcassets/placeholder.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "placeholder.png",
```

Comment:
Placeholder.png being transparent is something we will have to be careful of when using. Anywhere that it could be used, would require the .background to be set
Not something you need to change, just to be aware of!

Reply:
its not transparent